LICENSES-3rdparty.csv

```diff
@@ -1,288 +1,312 @@
-escodegen,npm,BSD-2-Clause,Yusuke Suzuki <utatane.tea@gmail.com>
-eslint,npm,MIT,Nicholas C. Zakas <nicholas+npm@nczconsulting.com>
+escalade,npm,MIT,Luke Edwards (https://lukeed.com)
+escape-string-regexp,npm,MIT,Sindre Sorhus (sindresorhus.com)
+escodegen,npm,BSD-2-Clause,(http://github.com/estools/escodegen)
+eslint,npm,MIT,Nicholas C. Zakas (https://eslint.org)
 eslint-config-prettier,virtual,MIT,Simon Lydell
-eslint-import-resolver-node,npm,MIT,Ben Mosher (me@benmosher.com)
-eslint-module-utils,npm,MIT,Ben Mosher <me@benmosher.com>
-eslint-plugin,virtual,MIT,
-eslint-plugin-import,virtual,MIT,Ben Mosher <me@benmosher.com>
-eslint-plugin-prettier,virtual,MIT,Teddy Katz
-eslint-scope,npm,BSD-2-Clause,
-eslint-utils,npm,MIT,Toru Nagashima
-eslint-visitor-keys,npm,Apache-2.0,Toru Nagashima (https://github.com/mysticatea)
-espree,npm,BSD-2-Clause,Nicholas C. Zakas <nicholas+npm@nczconsulting.com>
-esprima,npm,BSD-2-Clause,Ariya Hidayat <ariya.hidayat@gmail.com>
-esquery,npm,BSD-3-Clause,Joel Feenstra <jrfeenst+esquery@gmail.com>
-esrecurse,npm,BSD-2-Clause,Yusuke Suzuki <utatane.tea@gmail.com>
-estraverse,npm,BSD-2-Clause,Yusuke Suzuki <utatane.tea@gmail.com>
-estree,npm,MIT,
-esutils,npm,BSD-2-Clause,Yusuke Suzuki <utatane.tea@gmail.com>
-events,npm,MIT,Irakli Gozalishvili <rfobic@gmail.com> (http://jeditoolkit.com)
-exec-sh,npm,MIT,Aleksandr Tsertkov <tsertkov@gmail.com>
-execa,npm,MIT,Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)
-exit,npm,MIT,Cowboy Ben Alman (http://benalman.com/)
-expand-brackets,npm,MIT,Jon Schlinkert (https://github.com/jonschlinkert)
+eslint-import-resolver-node,npm,MIT,Ben Mosher (https://github.com/benmosher/eslint-plugin-import)
+eslint-module-utils,virtual,MIT,Ben Mosher (https://github.com/benmosher/eslint-plugin-import#readme)
+eslint-plugin-import,virtual,MIT,Ben Mosher (https://github.com/benmosher/eslint-plugin-import)
+eslint-plugin-prettier,virtual,MIT,Teddy Katz (https://github.com/prettier/eslint-plugin-prettier#readme)
+eslint-scope,npm,BSD-2-Clause,(http://github.com/eslint/eslint-scope)
+eslint-utils,npm,MIT,Toru Nagashima (https://github.com/mysticatea/eslint-utils#readme)
+eslint-visitor-keys,npm,Apache-2.0,Toru Nagashima (https://github.com/eslint/eslint-visitor-keys#readme)
+espree,npm,BSD-2-Clause,Nicholas C. Zakas (https://github.com/eslint/espree)
+esprima,npm,BSD-2-Clause,Ariya Hidayat (http://esprima.org)
+esquery,npm,BSD-3-Clause,Joel Feenstra (https://github.com/estools/esquery/)
+esrecurse,npm,BSD-2-Clause,(https://github.com/estools/esrecurse)
+estraverse,npm,BSD-2-Clause,(https://github.com/estools/estraverse)
+esutils,npm,BSD-2-Clause,(https://github.com/estools/esutils)
+events,npm,MIT,Irakli Gozalishvili (http://jeditoolkit.com)
+evp_bytestokey,npm,MIT,Calvin Metcalf (https://github.com/crypto-browserify/EVP_BytesToKey)
+exec-sh,npm,MIT,Aleksandr Tsertkov
+execa,npm,MIT,Sindre Sorhus (sindresorhus.com)
+exit,npm,MIT,"Cowboy" Ben Alman (https://github.com/cowboy/node-exit)
```

Comment:
The [parsing](https://github.com/DataDog/build-plugin/blob/87f850209a4c1a2630e9c31eda07f069f08d3cee/LICENSES-3rdparty.csv) seems to fail on this line, maybe because of `"`.